package.json

```diff
@@ -15,68 +15,70 @@
     "build": "NODE_ENV=production node scripts/build.js",
     "make": "export NODE_ENV=production && node scripts/build.js && electron-forge make -p 'darwin'",
     "make-windows": ".\\node_modules\\.bin\\electron-rebuild.cmd && set NODE_ENV=production&& node scripts/build.js && electron-forge make -p win32",
-    "make-linux": "export NODE_ENV=production && node scripts/build.js && electron-forge make -p 'linux'"
+    "make-linux": "export NODE_ENV=production && node scripts/build.js && electron-forge make -p 'linux'",
+    "preinstall": "cd BitGoJS/modules/core && npm i && npm run compile-dbg && cp ./dist/browser/BitGoJS.js ../../../public/BitGoJS.js && cd ../../.."
```

Comment:
We don't need this anymore after bitgo version 6 - the browser bundle is included in dist/browser/BitGoJS.min.js. However, this is the production, minified version. If you want to bundle the debug version you'll have to continue doing this.